devito/petsc/types/object.py

```diff
@@ -15,43 +23,37 @@ class DM(LocalObject):
     """
     dtype = CustomDtype('DM')
 
-    def __init__(self, *args, stencil_width=None, **kwargs):
+    def __init__(self, *args, dofs=1, **kwargs):
         super().__init__(*args, **kwargs)
-        self._stencil_width = stencil_width
+        self._dofs = dofs
 
     @property
-    def stencil_width(self):
-        return self._stencil_width
+    def dofs(self):
+        return self._dofs
 
     @property
     def _C_free(self):
         return petsc_call('DMDestroy', [Byref(self.function)])
 
     @property
     def _C_free_priority(self):
-        return 3
+        return 4
```

Comment:
:raised_eyebrow: 

https://xkcd.com/221/

Reply:
yes, we need an enumeration or something, because what started off as a simple thing is now growing out of hand :)